src/depends.py

```diff
@@ -1,15 +1,17 @@
-#! python
+#!/usr/bin/env python2
 
-import sys
 import os
 import pyelliptic.openssl
+import sys
 
 # Only really old versions of Python don't have sys.hexversion. We don't
 # support them. The logging module was introduced in Python 2.3
 if not hasattr(sys, 'hexversion') or sys.hexversion < 0x20300F0:
-    sys.stdout.write('Python version: ' + sys.version)
-    sys.stdout.write('PyBitmessage requires Python 2.7.3 or greater (but not Python 3)')
-    sys.exit()
+    sys.exit(
+        'Python version: %s\n'
+        'PyBitmessage requires Python 2.7.3 or greater (but not Python 3)'
+        % sys.version
+    )
 
 # We can now use logging so set up a simple configuration
 import logging
```

Comment:
why can't we have this import in the starting?

Reply:
Because this module tries to import each dependency separately and exit early. @PeterSurda, maybe you can explain how it was designed?

Reply:
@omkar1117 this file is for verifying deployment environment so the imports happen after the checks